README.md

```diff
@@ -52,3 +52,139 @@ For a basic setup allowing you to build the API and run tests:
 ## API Reference
 
 See the [Vega-Lite JavaScript API Reference](https://vega.github.io/vega-lite-api/api).
+
+## Usage
+
+### vega-lite API For Observable Notebooks
+
+Just import it like this:
+
+~~~js
+import {vl} from '@vega/vega-lite-api'
+~~~
+
+### vega-lite API for Browsers
+
+To use the vega-lite API on a browser, you need to include all the dependencies, set the default configuration and finally register it. Here is some starting code you can build from
+
+~~~html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+  </head>
+  <body>
+    <div id="chart"></div>
+
+    <script src="https://cdn.jsdelivr.net/npm/vega"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite-api"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-tooltip"></script>
+
+    <script>
+      const options = {
+        config: {
+          // vega-lite default configuration
+        },
+        init: (view) => {
+          // initialize tooltip handler
+          view.tooltip(new vegaTooltip.Handler().call);
+          // enable horizontal scrolling for large plots
+          if (view.container()) view.container().style["overflow-x"] = "auto";
+        },
+        view: {
+          // view constructor options
+          loader: vega.loader({
+            baseURL: "https://cdn.jsdelivr.net/npm/vega-datasets@1/",
+          }),
+          renderer: "canvas",
+        },
+      };
+
+      vl.register(vega, vegaLite, options);
+
+      vl.markBar({ tooltip: true })
+        .data([
+          { a: "A", b: 28 },
+          { a: "B", b: 55 },
+          { a: "C", b: 43 },
+          { a: "D", b: 91 },
+          { a: "E", b: 81 },
+          { a: "F", b: 53 },
+          { a: "G", b: 19 },
+          { a: "H", b: 87 },
+          { a: "I", b: 52 },
+        ])
+        .encode(
+          vl.x().fieldQ("b"),
+          vl.y().fieldN("a"),
+          vl.tooltip([vl.fieldQ("b"), vl.fieldN("a")])
+        )
+        .render()
+        .then((chart) => {
+          document.getElementById("chart").appendChild(chart);
+        });
+    </script>
+  </body>
+</html>
+~~~
+
+
+### vega-lite API For Nodejs
+
+~~~
+npm install vega vega-lite vega-tooltip vega-lite-api 
+~~~
+
+then import everything set your options and register. Here is an example
+
+~~~js
+const vega = require("vega");
```

Comment:
In moderns JS< you can use `import vega from 'vea';` etc.

Reply:
Yes of course, however node LTS doesn't support ES6 modules out of the box. The easiest way of supporting ES6 modules would be to add the *esm* dependency and then run the code as:

~~~
node -r esm index.js
~~~

Is it worth it?

If so, then I found the way of importing the libraries would be:

~~~
import * as vega from "vega";
import * as vegaLite from "vega-lite";
import * as vegaTooltip from "vega-tooltip";
import * as vl from "vega-lite-api";
~~~

Reply:
I wonder whether we are adding more confusion by adding the example than we are helping. I like the browser examples but would maybe cut the node example.

Reply:
It depends how much you want to encourage people to use it on node. I will start by agreeing that vl on node is a bit of a long stretch. Maybe useful for server side rendering, but that's it.

Having said that I feel like the Vega-Lite API is a very big step forward for the whole Vega ecosystem, but I it's losing its appealing with the very step barrier of entering. Using it in anything other than Observable is very difficult. Having these examples here should help more than hurt...

Reply:
+1, been struggling to get background/ticks etc to custom colors for several hours now :D